gateway/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kaazing/enterprise-gateway:5.1.5
+FROM kaazing/development-enterprise-gateway:5.2.1-RC003
```

Comment:
The problem is that the latest public image doesn't work. So we need to wait for a newer version of the gateway.